docs/namespace.md

```diff
@@ -1,72 +1,25 @@
 # Run in custom namespace
 
-In Velero version 0.7.0 and later, you can run Velero in any namespace. To do so, you specify the
-namespace in the YAML files that configure the Velero server. You then also specify the namespace when
-you run Velero client commands.
+In Velero version 0.7.0 and later, you can run Velero in any namespace.
 
-## Edit the example files
-
-The Velero release tarballs include a set of example configs that you can use to set up your Velero server. The
-examples place the server and backup/schedule/restore/etc. data in the `velero` namespace.
-
-To run the server in another namespace, you edit the relevant files, changing `velero` to
-your desired namespace.
-
-To store your backups, schedules, restores, and config in another namespace, you edit the relevant
-files, changing `velero` to your desired namespace. You also need to create the
-`cloud-credentials` secret in your desired namespace.
+To store your backups, schedules, restores, and config in another namespace, you run the install command with the `--namespace` parameter.
```

Comment:
I'd probably just drop this sentence -- it's redundant and slightly misleading, since the `--namespace` flag will *also* deploy the Velero server into the specified namespace